Packages/Common/DashlaneAppKit/Sources/DashlaneAppKit/SessionServices/SyncService.swift

```diff
@@ -175,7 +175,7 @@ public class SyncService: Mockable {
     }
     
             public func sync(triggeredBy trigger: Definition.Trigger) {
-        syncLogger.debug("dipatching a sync")
+        syncLogger.debug("dispatching a sync")
```

Comment:
fwiw, i'd personally use `dispatching a sync request` -- `a sync` looks like a typo for `async`...